tdrs-backend/tdpservice/scheduling/parser_task.py

```diff
@@ -37,10 +37,11 @@ def parse(data_file_id):
 
     logger.info(f"Parsing finished for file -> {repr(data_file)} with status {dfs.status} and {len(errors)} errors.")
 
-    recipients = User.objects.filter(
-        stt=data_file.stt,
-        account_approval_status=AccountApprovalStatusChoices.APPROVED,
-        groups=Group.objects.get(name='Data Analyst')
-    ).values_list('username', flat=True).distinct()
+    if should_send_submission_email:
```

Comment:
Minor nit, but to be pythonic we should have an explicit comparison such as `if should_send_submission_email is True:`. In the past we've been bit using an implicit conditional evaluating when an edge case or error presented itself.